### PR TITLE
VAULT_API incorrect for hvac

### PR DESCRIPTION
**Pull request by bobby-didcoding**
This PR is to fix the login to vault

### CODE REVIEW

File: bootstrap.py [hunk 1]
```diff
@@ -16,12 +16,12 @@ def __init__(self):
     def client(self):
 
       # Create a client instance
-      client = hvac.Client(url=self.VAULT_API)
+      client = hvac.Client(url='https://vault.ci.uktrade.digital')
```
Comment (ecbm) [inline bootstrap.py:19]: Why are we changing this to be hard coded?

Comment (bobby-didcoding) [reply bootstrap.py:19]: This will be removed when I have tested the auth funtionality which I can't do locally. I'll streamline the script when It works.

Comment (bobby-didcoding) [reply bootstrap.py]: @ecbm I have changed the script to alter the given url as hvac appends v1 by default. There is not need for the hard coded url anymore :)

**Review by ecbm — APPROVED** [review]
lgtm!

**Pull request merged**